tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -891,9 +911,13 @@ _prevctx = localctx
 >>
 
 
-LexerFile(lexerFile, lexer, namedActions) ::= <<
-<fileHeader(lexerFile.grammarFileName, lexerFile.ANTLRVersion)>
-package parser
+LexerFile(file, lexer, namedActions) ::= <<
+<fileHeader(file.grammarFileName, file.ANTLRVersion)>
+<if(file.genPackage)>
+package <file.genPackage><! // <file.grammarName>!><! why can't we access file.grammarName here? !>
```

Comment:
Isn't file.grammarName usable in the other file templates? What goes wrong here when we try to copy what those templates do?